openmetadata-spec/src/main/resources/json/schema/analytics/webAnalyticEventType/customEvent.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$id": "https://open-metadata.org/schema/analytics/customEvent.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "pageViewData",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.analytics.customEvent",
+  "description": "Event tracker (e.g. clicks, etc.)",
+  "definitions": {
+    "customEventTypes": {
+      "description": "Type of events that can be performed",
+      "type": "string",
+      "enum": [
+        "CLICK"
+      ]
+    }
+  },
+  "properties": {
+        "fullUrl": {
+          "description": "complete URL of the page",
+          "type": "string"
+        },
+        "url": {
+          "description": "url part after the domain specification",
+          "type": "string"
+        },
+        "hostname": {
+          "description": "domain name",
+          "type": "string"
+        },
+        "sessionId": {
+          "description": "Unique ID identifying a session",
+          "$ref": "../../type/basic.json#/definitions/uuid"
+        },
+        "eventType": {
+          "description": "Type of event that was performed",
+          "$ref": "#/definitions/customEventTypes"
+        },
+        "eventValue": {
+            "description": "Value of the event",
+            "type": "string"
+            }
+        }
```

Comment:
nit: 2 spaces for indenting

Reply:
I think there are common properties with `PageViewEvent.json` that could be part of `webAnalyticEventData.json` like fullUrl, url, hostname, sessionId, etc.